xbmc/windowing/Resolution.cpp

```diff
@@ -20,6 +20,12 @@
 
 #include <cstdlib>
 
+EdgeInsets::EdgeInsets() = default;
```

Comment:
could be done in the header file

Reply:
I had an impression that adding definitions to the header is bad practice. Or trivial ones are ok? Will change.

Also a question: is `float left{0.0f};` or even `float left{};` better than `float left = 0.0f;`?

Reply:
@Rechi please check my question above

Reply:
IMO adding `= default` for constructors is okay in the header file.
Whatever you prefer.

Reply:
thanks!